boards/arm/stm32wb5mm_dk/doc/stm32wb5mm_dk.rst

```diff
@@ -0,0 +1,226 @@
+microcontroller,STM32WB5MMG provides best-in-class RF performance thanks
+to its high receiver sensitivity and output power signal. Its low-power
+features enable extended battery life, small coin-cell batteries, and
+energy harvesting.
+
+- Ultra-low-power with FlexPowerControl
+- Core: ARM |reg| 32-bit Cortex |reg|-M4 CPU with FPU
+- Radio:
+- 2.4GHz
+
+  - RF transceiver supporting:
+
+    - Bluetooth |reg| 5.4 specification,
+    - IEEE 802.15.4-2011 PHY and MAC,
+    - Zigbee|reg| 3.0
+  - RX sensitivity:
+
+    - -96 dBm (Bluetooth |reg| Low Energy at 1 Mbps),
+    - -100 dBm (802.15.4)
+
+  - Programmable output power up to +6 dBm with 1 dB steps
+  - Integrated balun to reduce BOM
+  - Support for 2 Mbps
+  - Support GATT caching
+  - Support EATT (enhanced ATT)
+  - Support advertising extension
+  - Accurate RSSI to enable power control
+
+- Clock Sources:
+
+  - 32 MHz crystal oscillator with integrated
+    trimming capacitors (Radio and CPU clock)
+  - 32 kHz crystal oscillator for RTC (LSE)
+  - Internal low-power 32 kHz (±5%) RC (LSI1)
+  - Internal low-power 32 kHz (stability
+    ±500 ppm) RC (LSI2)
+  - Internal multispeed 100 kHz to 48 MHz
+    oscillator, auto-trimmed by LSE (better than
+    ±0.25% accuracy)
+  - High speed internal 16 MHz factory
+    trimmed RC (±1%)
+  - 2x PLL for system clock, USB, SAI, ADC
+
+More information about STM32WB55RG can be found here:
+
+- `STM32WB5MM-DK on www.st.com`_
+- `STM32WB5MMG datasheet`_
+
+Supported Features
+==================
+
+The Zephyr STM32WB5MM-DK board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+
+
+Other hardware features are not yet supported on this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+``boards/arm/stm32wb5mm_dk/stm32wb5mm_dk_defconfig``
+
+Bluetooth and compatibility with STM32WB Copro Wireless Binaries
+================================================================
+
+To operate bluetooth on STM32WB5MMG, Cortex-M0 core should be flashed with
+a valid STM32WB Coprocessor binaries (either 'Full stack' or 'HCI Layer').
+These binaries are delivered in STM32WB Cube packages, under
+Projects/STM32WB_Copro_Wireless_Binaries/STM32WB5x/
+For compatibility information with the various versions of these binaries,
+please check `modules/hal/stm32/lib/stm32wb/hci/README`_
+in the hal_stm32 repo.
+Note that since STM32WB Cube package V1.13.2, "full stack" binaries are not compatible
+anymore for a use in Zephyr and only "HCI Only" versions should be used on the M0
+side.
```

Comment:
this could use empty lines to not render as a big, hard to read, block

![Uploading image.png…]()